spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -2,16 +2,17 @@
 
 require 'rails_helper'
 
-RSpec.describe Api::V1::KeywordsController, type: :controller do
+RSpec.describe Api::V1::KeywordsController, type: :request do
   describe 'GET#index' do
-    it 'returns expected status' do
-      get :index
+    it 'returns success status' do
+      get '/api/v1/keywords'
```

Comment:
I am not able to get this working 🤔 Perhaps the project is missing some plugin? 
<img width="572" alt="Screen Shot 2023-06-06 at 17 10 35" src="https://github.com/nimblehq/ic-rails-phong-bliss/assets/6356137/7172c18f-0faa-4d8a-90ac-93bd86642e8c">

Reply:
Working now with `api_v1_keywords_path` thank you for the suggestions and the link.

Reply:
Do you think it should work with `private_path` and `public_path` as well? I tried but failed 🤔

Reply:
`private` and `public` paths do not follow Rails standards — that also a reason why I recommend using real controllers over "temporary private/public" ones. 

The route is declared as a `resources route` but `private` and `public` are singular words. If you want them to work, you need to follow the Rails conventions:

- Pluralized controller name: `privates_controller.rb`
- Pluralized resources route: `resources :privates, only: :index`

Only from that Rails will be able to reflect the expected names (as far as I remember a model class is not mandatory — but to be confirmed). 


On the "idiomatic" side, `private` and `public` are not resource names (you will never have a DB table called `privates` or `publics`). If you want to follow the conventions, it might be better to choose a name like "private_item" (with "private_items_controller.rb" / and the route `resources :private_items`).

Reply:
this is the "weird", "nice" and sometimes "annoying" part of Rails :D 

This famous "convention over configuration": If you follow the conventions, a lot of magic will happen (e.g. `_path` short hands, routes, model reflextion, etc...) but it means we have to be very careful in learning and applying those conventions (via Rails guides mainly) ;-)

Reply:
Thank you very much. I am getting started to understand Ruby's magic. Updating this PR to make it more confirming to the conventions 🙏